resources/Materials/TestSuite/_options.mtlx

```diff
@@ -105,5 +105,10 @@
       <parameter name="wedgeRangeMax" type="floatarray" value="1.0, 0.9" />
       <parameter name="wedgeSteps" type="integerarray" value="4, 3" />
 
+      <!-- Bake rendering options: Currently only suppoers the genglsl_glsl400 shader generator -->
+      <parameter name="bakeFiles" type="string" value="mapped_surfaceshader.mtlx" />
```

Comment:
You can pick which files to bake. I have not done extensive testing on this but the same file has color bake + unit baking.